src/App.tsx

```diff
@@ -21,6 +33,11 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={itemsPerPage}
+            onChange={(event) => {
+              setItemsPerPage(+event.target.value);
+              onPageChange(1);
```

Comment:
Replace `+event.target.value` with `Number(event.target.value)` for better readability.